### PR TITLE
Added the option of violence to a few situations.

### PR DESCRIPTION
**Pull request by dela0130**
Hey guys, what do you think? This is the Wild West after all.

### CODE REVIEW

File: conman.md [hunk 1]
```diff
@@ -26,4 +26,6 @@ The whole bar was watching in awe.
 
 "A deal is a deal." 
 
-Bart and the other man are holding him and dragging him out by his arms while Tom shamelessly thrashes about.
```
Comment (dela0130) [inline conman.md:29]: @tran0463 This is how you can comment in line!!

What do you think about me adding this line to the end of your story?

File: conman.md [hunk 2]
```diff
@@ -26,4 +26,6 @@ The whole bar was watching in awe.
...
\ No newline at end of file
+Bart and the other man are holding him and dragging him out by his arms while Tom shamelessly thrashes about.
+
```
Comment (dela0130) [inline conman.md:30]: @tran0463 Maybe it's HERE that I should have commented about whether or not you like this line. 🤔

**Review by laal0007 — COMMENTED** [review]
Oh yes -- this is definitely a realistic story direction. They are more likely to get violent, so I agree with the additional options.

**Comment by fbasarn** [comment]
Reviewed! I also agree, thought the story would get violent eventually considering it's a cowboy story

**Comment by tran0463** [comment]
Some pretty funny links. Looks good

**Pull request merged**